packages/engine/src/FeatureFlagsState.tsx

```diff
@@ -37,7 +37,7 @@ export const FeatureFlagsState = defineState({
   },
   useEnabled(flagName: FeatureFlag) {
     const state = useHookstate(getMutableState(FeatureFlagsState)[flagName]).value
-    return typeof state === 'boolean' ? state : true
+    return typeof state === 'boolean' ? state : typeof state === 'number' && state === 0 ? false : true
```

Comment:
When would this equal a number?

Reply:
Although at the schema level it is of type boolean, in the database (MariaDB) it is represented as 0 and 1 instead of false and true respectively, which is what the `state` value number is returning. I don't see any unit test for this, and that's what I found when I was debugging.

Reply:
![Screenshot 2024-06-26 at 6 52 52 a m](https://github.com/EtherealEngine/etherealengine/assets/12107518/03f68f79-2f93-4e27-b11b-21b4b1356486)
i tested out modifying directly in the database and is working considering that, because in other hand always return true because the type of `state` isn't boolean

Reply:
@kattsushi I think you need a resolver for flagValue like https://github.com/EtherealEngine/etherealengine/blob/4ea6942fc0f39af7dbeacd195942a0b9ca1deaa8/packages/server-core/src/social/location/location.resolvers.ts#L80

Reply:
thanks @hanzlamateen fixed